app/scripts/OverlayTrack.js

```diff
@@ -11,7 +11,130 @@ class OverlayTrack extends PixiTrack {
     this.drawnRects = {};
   }
 
-  drawHorizontalOverlay(graphics, position, extent, minWidth = 0) {
+  drawWithPositionedBorder(
+    graphics,
+    xPos,
+    yPos,
+    width,
+    height,
+    fill,
+    stroke,
+    outline,
+    isVertical = false
+  ) {
+    let finalXPos = xPos;
+    let finalYPos = yPos;
+    let finalWidth = width;
+    let finalHeight = height;
+
+    const outlineWidth = outline.width * 2 > width
+      ? width / 2 : outline.width;
+    const outlineHeight = outline.width * 2 > height
```

Comment:
Is this right? Should it be `outline.height * 2 > height`?

In either case, a comment above these lines describing what they're trying to accomplish would be helpful.

Reply:
Agreed, I'll add a comment. Apart from that this is correct because `outline.height` doesn't exist. The size of the stroke and outline is one dimensional so width == height.